Makefile

```diff
@@ -3,13 +3,13 @@ ifeq (,$(shell which tt 2>/dev/null))
 	TTCTL := tarantoolctl
 endif
 
-.rocks: cartridge-metrics-role-scm-1.rockspec
-	$(TTCTL) rocks make
+.rocks:
```

Comment:
`cartridge-metrics-role-scm-1.rockspec` and `cartridge/*` should be here

Reply:
Returned as it was